ome_zarr/writer.py

```diff
@@ -172,28 +189,32 @@ def write_multiscale(
     axes: str or list of str or list of dict
       List of axes dicts, or names. Not needed for v0.1 or v0.2
       or if 2D. Otherwise this must be provided
-    transformations: 2Dlist of dict
+    coordinateTransformations: 2Dlist of dict
       For each path, we have a List of transformation Dicts (not validated).
       Each list of dicts are added to each datasets in order.
     """
 
     dims = len(pyramid[0].shape)
     axes = _get_valid_axes(dims, axes, fmt)
 
-    paths = []
+    datasets: List[Union[str, dict]] = []
     for path, dataset in enumerate(pyramid):
         # TODO: chunks here could be different per layer
         group.create_dataset(str(path), data=dataset, chunks=chunks)
-        paths.append(str(path))
-    write_multiscales_metadata(group, paths, fmt, axes, transformations)
+        datasets.append({"path": str(path)})
```

Comment:
Should a default `scale` transformation be computed and stored under `"coordinateTransformations"` at this stage to comply with the latest specification proposal?

If so, what should be the behavior if an additional `coordinateTransformations` dictionary is passed?

Reply:
If a `scale` is provided for each level, then we just use that. If ONLY a `translation` is provided, then I guess we can append that to the default `scale`